pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cyclonedx-bom"
-version = "3.11.0"
+version = "4.0.0"
```

Comment:
please revert this change.

the version here is managed by the release process, which will increment this value on purpose.